blueoil/utils/config.py

```diff
@@ -67,6 +67,11 @@ def _saved_config_file_path():
         if os.path.isfile(filepath):
             return filepath
 
+    raise FileNotFoundError("Config file not found: '{}' nor '{}'".format(
```

Comment:
can you use the same list in the for loop above and here?

Reply:
Thank you! Fixed.